### PR TITLE
Optimize functions to improve code readability

### PR DESCRIPTION
**Pull request by tangming1996**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Improve code readability

### CODE REVIEW

**Comment by Shelley-BaoYue** [comment]
Thank u for contributions!  Please provide a more accurate description in the pr title.

**Comment by tangming1996** [comment]
> Thank u for contributions! Please provide a more accurate description in the pr title.

Thank you for your suggestion, the revision has been submitted

File: cloud/pkg/controllermanager/edgeapplication/overridemanager/imageparser/imageparser.go [hunk 1]
```diff
@@ -153,14 +153,16 @@ func Parse(image string) (*Components, error) {
 	}
 
 	comp := &Components{}
-	if named, ok := ref.(reference.Named); ok {
+	switch named := ref.(type) {
+	case reference.Named:
 		comp.hostname, comp.repository = SplitHostname(named.Name())
 	}
```
Comment (fisherxu) [inline cloud/pkg/controllermanager/edgeapplication/overridemanager/imageparser/imageparser.go:159]: Only one case, seems have no difference with if..

Comment (tangming1996) [reply cloud/pkg/controllermanager/edgeapplication/overridemanager/imageparser/imageparser.go]: Yes, if it's just if, there's really no need to use the switch statement, I've changed it.

**Pull request merged**